dashboard/modules/reporter/tests/test_reporter.py

```diff
@@ -629,7 +628,7 @@ def test_enable_k8s_disk_usage(enable_k8s_disk_usage: bool):
             assert root_usage.free == 1
 
 
-def test_reporter_worker_cpu_percent():
+def test_reporter_worker_cpu_percent(shutdown_only):
```

Comment:
this is not needed.

Reply:
Thanks, I forgot to delete this function params. I double checked again and delete all the unnecessary shutdown_only